src/app/component/quiz/quiz.component.html

```diff
@@ -19,21 +19,16 @@
                 <div class="m-3">
                   <div class="card quiz-card h-auto">
                     <div
-                      class="card-header header-txt py-0 px-3 d-flex align-items-center justify-content-between position-relative"
+                      class="card-header header-txt py-3 px-3 d-flex align-items-center justify-content-between position-relative"
                     >
```

Comment:
instead of giving px-3 and py-3 individually replace it with p-3

Reply:
ok, i changed it.